javav2/example_code/lambda/src/main/java/com/example/lambda/LambdaScenario.java

```diff
@@ -0,0 +1,259 @@
+// snippet-sourcedescription:[LambdaScenario.java demonstrates how to perform various operations by using the LambdaClient object.]
+//snippet-keyword:[AWS SDK for Java v2]
+// snippet-keyword:[AWS Lambda]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[04/14/2022]
+// snippet-sourceauthor:[AWS-scmacdon]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.lambda;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.lambda.LambdaClient;
+import software.amazon.awssdk.services.lambda.model.*;
+import software.amazon.awssdk.services.lambda.model.UpdateFunctionConfigurationRequest;
+import software.amazon.awssdk.services.lambda.model.Runtime;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/*
+ *  Function names appear as arn:aws:lambda:us-west-2:335556666777:function:HelloFunction
+ *  you can retrieve the value by looking at the function in the AWS Console
+ *
+ *  Before running this Java code example, set up your development environment, including your credentials.
+ *
+ *  For more information, see this documentation topic:
+ *
+ *  https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ *
+ *  This example performs these tasks.

```

Comment:
This example performs the following tasks:

Reply:
fixed